vars/kola.groovy

```diff
@@ -102,6 +103,13 @@ def call(params = [:]) {
             args += " --exttest=${env.WORKSPACE}/${path}"
         }
 
+        def skipKolaTags = params.get('skipKolaTags', "")
+        def skipKolaTagsArgs = ""
```

Comment:
How about instead of this new variable, we just keep modifying `args` like we do for other directives?

Reply:
I had thought about that, but we `args` isnt passed to all of the `kola()` runs. Now that we passing this  to all of them, I'll simplify it and just append to `args` :+1: